app/src/main/java/com/lightricks/feedexercise/ui/feed/FeedFragment.kt

```diff
@@ -25,18 +29,32 @@ class FeedFragment : Fragment() {
     private lateinit var dataBinding: FeedFragmentBinding
     private lateinit var viewModel: FeedViewModel
     private lateinit var feedAdapter: FeedAdapter
+    private lateinit var feedRepository: FeedRepository
```

Comment:
I don't see a reason for holding the repository instance here. 

What's the reason the fragment is keeping a reference to the repository?